### PR TITLE
Updated README.md with roadmap information

### PR DESCRIPTION
**Pull request by EkinKarabulut**
Short-term and long-term roadmaps are included

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,45 @@ Integration with JupyterLab [(Take me to the installation guide!)](https://githu
...
+
+After gathering valuable feedback and comments from data scientists, we created the following short-term roadmap for genv:
+
+**1. Environment specific information:** With this feature, you will be able to see the processes, machines and utilization of the machines specifically to your GPU environment when you type nvidia-smi
+
+**2. Docker support:** Running Docker containers powered by the GPUs in your environment
+
+**3. Configuring GPU memory:** Specifying the GPU memory that needs to be configured in an environment
+
+**4. To be able to provisioning the GPU fractions**
```
Comment (razrotenberg) [inline README.md:59]: I think it's more correct to say: "To be able to provision fractions of GPUs"
wdyt?

File: README.md [hunk 2]
```diff
@@ -44,6 +44,45 @@ Integration with JupyterLab [(Take me to the installation guide!)](https://githu
 
 A PyCharm integration is also in our roadmap so stay tuned!
 
+## 💨 Guide to the Roadmap
+
+### **What is coming up soon?**
+
+After gathering valuable feedback and comments from data scientists, we created the following short-term roadmap for genv:
+
+**1. Environment specific information:** With this feature, you will be able to see the processes, machines and utilization of the machines specifically to your GPU environment when you type nvidia-smi
```
Comment (razrotenberg) [inline README.md:53]: The truth is that we won't show utilization specific for the environment so I'd omit this part of the utilization

**Review by razrotenberg — APPROVED** [review]
It's great ! 😀 👏

File: README.md (hunk 1 above)
Comment (EkinKarabulut) [reply README.md:59]: hahahaha yes you are right

**Pull request merged**